crates/pallet-runtime-configs/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2022 Subspace Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Pallet for tweaking the runtime configs for multiple network.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+mod pallet {
+    use frame_support::pallet_prelude::*;
+    use frame_system::pallet_prelude::*;
+
+    #[pallet::pallet]
+    #[pallet::generate_store(pub trait Store)]
+    pub struct Pallet<T>(_);
+
+    /// Enable the signed extension `DisablePallets`.
+    #[pallet::storage]
+    #[pallet::getter(fn enable_disable_pallets)]
+    pub type EnableDisablePallets<T> = StorageValue<_, bool, ValueQuery>;
```

Comment:
This is sort of counter-intuitive, I'd like to rename the signed extension `DisablePallets` to something `CheckCall`, what do you think? @nazar-pc

Reply:
Signed extension can be left as is, but this parameter should be named something like `EnableExecutor` and default to `false` in current Gemini